.azure/templates/run-spinquic.yml

```diff
@@ -7,6 +7,7 @@ parameters:
   config: 'Debug'
   arch: 'x64'
   tls: ''
+  allocFail: 1000
```

Comment:
Maybe make this 1 in a 100 instead of 1000?

Reply:
I'll do that in a follow up PR which enables this.